app.rb

```diff
@@ -0,0 +1,18 @@
+require 'rubygems'

```

Comment:
Carriage return character detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.